runtime/port/CMakeLists.txt

```diff
@@ -31,33 +31,57 @@
 
 
 set(OBJECTS "")
-list(APPEND OBJECTS 
+list(APPEND OBJECTS
+	j9gp.c
 	j9hypervisor.c
 	j9hypervisor_common.c
 	j9ipcmutex.c
-	j9SysvIPCWrappers.c
 	j9port.c
 	j9process.c
 	j9gs.c
-	j9gp.c
-	j9sharedhelper.c
 	j9shmem.c
 	j9shsem.c
 	j9shsem_deprecated.c
 	j9sock.c
 	j9sockptb.c
 	j9sysinfo.c
-	j9sysinfo_helpers.c
 	j9mem_basic.c
 	j9portcontrol.c
 )
 if(OMR_ARCH_X86)
-	list(APPEND OBJECTS
-		j9hypervisor_i386.c
-		j9sysinfo_helpers.c
-	)
+	list(APPEND OBJECTS j9hypervisor_i386.c)
 elseif(OMR_ARCH_POWER)
 	list(APPEND OBJECTS j9hypervisor_systemp.c)
+elseif(OMR_ARCH_S390)
+	list(APPEND OBJECTS
+		j9hypervisor_systemz.c
+		j9gs_s390.c
+	)
+endif()
+
+if(OMR_ARCH_X86 OR OMR_ARCH_S390)
```

Comment:
The equivalent to this in module.xml is:
https://github.com/eclipse/openj9/blob/e2c4592f1b2ab0fcdca3d76e2414593863c58a0c/runtime/port/module.xml#L131-L137

Does `OMR_ARCH_X86` cover linux_x86, win_x86, & osx?

Reply:
Yes, the `OMR_ARCH_XXX` flags are independent of os.